docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -0,0 +1,81 @@
+---
+description: To prevent "resource not accessible by integration" error when running job to upload findings to GitHub's Advanced Security Dashboard
+tags:
+  - Semgrep CI
+  - GitHub
+---
+
+# Uploading findings to GitHub Advanced Security Dashboard
+
```

Comment:
Hey @pjzcodes ! We use the Google Style Guide (I see that you're already following a lot of the conventions!). For headings, I'd just like to add that we don't leave headings empty, that is, headings without at least an introductory sentence. 

Citation: [https://developers.google.com/style/headings#heading-and-title-format](https://developers.google.com/style/headings#heading-and-title-format)

Screenshot: 
<img width="550" alt="Snag_3cfaeb8" src="https://github.com/returntocorp/semgrep-docs/assets/20766840/544868f8-258d-4f31-bbae-999427c1dbb4">